src/appkit.cs

```diff
@@ -17986,6 +17986,42 @@ interface NSComboBoxDelegate {
 		void SelectionIsChanging (NSNotification notification);
 	}
 
+	[BaseType (typeof(NSObject))]
+	[Model]
+	[Protocol]
+	interface NSTokenFieldCellDelegate {		
+		[Export ("tokenFieldCell:completionsForSubstring:indexOfToken:indexOfSelectedItem:")]
+		string [] GetCompletionStrings (NSTokenFieldCell tokenFieldCell, string substring, nint tokenIndex, ref nint selectedIndex);
```

Comment:
The header says:
```Each element in the array should be an NSString or an array of NSStrings.```

```// Each element in the array should be an NSString or an array of NSStrings.
// substring is the partial string that is being completed.  tokenIndex is the index of the token being completed.
// selectedIndex allows you to return by reference an index specifying which of the completions should be selected initially. 
// The default behavior is not to have any completions.
- (NSArray *)tokenFieldCell:(NSTokenFieldCell *)tokenFieldCell completionsForSubstring:(NSString *)substring indexOfToken:(NSInteger)tokenIndex indexOfSelectedItem:(NSInteger *)selectedIndex;
```

So we likely have to bind this as just NSArray unfortunately unless @dalexsoto or someone else on XI has a cute trick.

Reply:
Yup we need `NSArray` here